src/test/polymer/paper-toolbar-v1-to-v2_test.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+import * as path from 'path';
+import {Analyzer, FSUrlLoader} from 'polymer-analyzer';
+
+import {Linter} from '../../linter';
+import {registry} from '../../registry';
+import {applyEdits, makeParseLoader} from '../../warning';
+
+const fixtures_dir = path.join(__dirname, '..', '..', '..', 'test');
+
+const ruleId = 'paper-toolbar-v1-to-v2';
+
+suite(ruleId, () => {
+  let analyzer: Analyzer;
+  let linter: Linter;
+
+  setup(() => {
+    analyzer = new Analyzer({urlLoader: new FSUrlLoader(fixtures_dir)});
+    linter = new Linter(registry.getRules([ruleId]), analyzer);
+  });
+
+  test('Child elements without slots or a special class have `slot="top"` added.',
```

Comment:
I'll shorten these up since they don't match the style for all the other test names.

Reply:
Fixed.